jupyter-config.yml

```diff
@@ -5,9 +5,11 @@ jupyterhub:
   singleuser:
     image:
       name: rhodium/notebook
-      tag: 4444a717c6b4e10796a0a3ba2871f03b2613a890
+      tag: c9a90c7266d833d9e1775b8b57098e2f9fdf2d4b
     storage:
       capacity: 10Gi
+      dynamic:
+        storageClass: pd-ssd
```

Comment:
Is this going to migrate safely or do we need to take steps to preserve people's data?

Reply:
I think it won't do anything if there are already standard volumes existing. It will just affect new users and if we were to delete an old users volume. We could also just keep it standard and then at some point try migrating to the filestore instance?